target/linux/ramips/image/mt7621.mk

```diff
@@ -1616,6 +1627,17 @@ define Device/youku_yk-l2
 endef
 TARGET_DEVICES += youku_yk-l2
 
+define Device/yuncore_ax820
+  $(Device/dsa-migration)
+  IMAGE_SIZE := 7872k
```

Comment:
That's a boldly wrong copy-paste either for the partitions or the IMAGE_SIZE.

Reply:
You're right, somehow I even mis-calculated the wrong image size when checking, fixed.